TrkHitReco/src/TNTClusterer.cc

```diff
@@ -102,22 +108,38 @@ namespace mu2e
       clusters[ic].clearHits();
     }
 
+    int method = distMethod_;
     int ditime(int(maxHitdt_/tbin_));
     unsigned nchanged(0);
     for (size_t ihit=0;ihit<BkgHits.size();++ihit) {
 
       auto& hit  = BkgHits[ihit];
       const auto& chit = chcol[hit.chidx_];
 
+      float dhit(0.), dseed(0.), mindist(0.);
```

Comment:
Please move this if outside the loop.  It only needs to be executed once/cluster, not once/hit.